### ISSUE TITLE
text-overflow: ellipsis in table

### ISSUE THREAD

**Issue by mauroesposito** [Kozea/WeasyPrint#1112]
Hello,

I've seen some issue about management of text overflow in table, but for some reasons, in my case I can't find a concrete response to solve my problem.

I'm trying to do something that in html can be processed without problem:

![image](https://user-images.githubusercontent.com/8039989/81708403-7bf39e00-9471-11ea-96ac-7e49539e7880.png)

But when I try to do that trough print pdf, this is the result:

![image](https://user-images.githubusercontent.com/8039989/81708476-8877f680-9471-11ea-8145-32428bd5740f.png)

Here styles used in my snippet:

```
body table.red_table tr .col1 {
    width: 58px;
    max-width: 58px;
}

body table.red_table tr th {
    color: #fff;
    background-color: #d85353;
    text-align: left;
    text-overflow: ellipsis;
    white-space: nowrap;
    overflow: hidden;
    border: none;
}
```

It's possible to have any tips on how is possible to solve this problem?

Thank you,
Mauro

**Comment by pinkynrg** [comment]
I have the same exact problem. Is there anyway this can be a bug?

**Comment by liZe** [comment]
Fixed by #2085.
